cmd/ddev/cmd/scripts/test_ddev.sh

```diff
@@ -105,37 +162,29 @@ ddev start -y || ( \
   ddev logs | tail -20l && \
   printf "============= contents of /mnt/ddev_config  =========\n" && \
   docker exec -it ddev-d9-db ls -l /mnt/ddev_config && \
-  printf "Start failed. Please provide this output in a new gist at gist.github.com\n" && \
+  printf "Start failed.\n" && \
   exit 1 )
 
-echo "======== Curl of site from inside container:"
+header "Curl of site from inside container"
 ddev exec curl --fail -I http://127.0.0.1
 
-echo "======== curl -I of http://${PROJECT_NAME}.ddev.site from outside:"
-curl --fail -I http://${PROJECT_NAME}.ddev.site
-if [ $? -ne 0 ]; then
-  set +x
-  echo "Unable to curl the requested project Please provide this output in a new gist at gist.github.com."
-  exit 1
-fi
-echo "======== full curl of http://${PROJECT_NAME}.ddev.site from outside:"
-curl http://${PROJECT_NAME}.ddev.site
+header "curl -I of http://${PROJECT_NAME}.ddev.site:${router_http_port} from outside"
+curl --fail -I http://${PROJECT_NAME}.ddev.site:${router_http_port}
+
+header "Full curl of http://${PROJECT_NAME}.ddev.site:${router_http_port} from outside"
+curl http://${PROJECT_NAME}.ddev.site:${router_http_port}
 
-echo "======== Project ownership on host:"
+header "Full curl of https://${PROJECT_NAME}.ddev.site:${router_https_port} from outside"
+curl https://${PROJECT_NAME}.ddev.site:${router_https_port}
+
+header "Project ownership on host"
 ls -ld ${PROJECT_DIR}
-echo "======== Project ownership in container:"
+header "Project ownership in container"
 ddev exec ls -ld /var/www/html
-echo "======== In-container filesystem:"
+header "In-container filesystem"
 ddev exec df -T /var/www/html
-echo "======== curl again of ${PROJECT_NAME} from host:"
-curl --fail http://${PROJECT_NAME}.ddev.site
-if [ $? -ne 0 ]; then
-  set +x
-  echo "Unable to curl the requested project Please provide this output in a new gist at gist.github.com."
-  exit 1
-fi
 
-echo "Thanks for running the diagnostic. It was successful."
-echo "Please provide the output of this script in a new gist at gist.github.com"
+header 'Thanks for running the diagnostic!'
 echo "Running ddev launch in 5 seconds" && sleep 5
 ddev launch
+ddev stop
```

Comment:
> ddev launch should not get a 404. Was that because of the router-http-port problem?

I get 404 because I use [braus](https://braus.properlypurple.com/) to open links.

This means, I have some delay after `ddev launch`, because I select the browser to open the website.

And right after `ddev launch` there is `ddev stop`.

Maybe add some `sleep` between the two commands? Because I may not be the only one who will be affected by this.

Reply:
thanks, that makes sense.